extension/aodn-classifier/src/main/java/au/org/emii/classifier/AodnThesaurus.java

```diff
@@ -13,17 +14,22 @@
  */
 
 public class AodnThesaurus {
+
+    private static Logger logger = Logger.getLogger(AodnThesaurus.class);
+
     private static final String LANG_CODE = "en";
     private static final Namespace SKOS_NAMESPACE = Namespace.getNamespace("skos","http://www.w3.org/2004/02/skos/core#");
     private static final Namespace DCTERMS_NAMESPACE = Namespace.getNamespace("dcterms","http://purl.org/dc/terms/");
 
     private final Thesaurus thesaurus;
 
-    public AodnThesaurus(Thesaurus thesaurus) {
+    public AodnThesaurus(Thesaurus thesaurus, String scheme) {
+        if (thesaurus == null) { logger.warn(String.format("Could not find thesaurus for scheme='%s'", scheme)); }
```

Comment:
Should this be logger.error ?

Reply:
At present there is no log4j config other than the default in use. (This might mean no logging?). So I guess no decision has yet been made what the default logging level will be?

I used this config in testing: 

```
log4j.rootLogger=DEBUG, stdout
log4j.appender.stdout=org.apache.log4j.ConsoleAppender
log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
log4j.appender.stdout.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
```

Reply:
There is current logging that can be seen in Sumo
EG: `_sourceCategory=prod/geonetwork/tomcat/catalina`

As to the log level... ? 
It wont be debug

Reply:
I chose warn level because the cause is not necessarily a problem in code but a problem in config/input. I will change it if required?